image/change.go

```diff
@@ -0,0 +1,283 @@
+// Copyright 2016 The Linux Foundation
```

Comment:
We probably want to revisit this header, especially if we're going to be copying large chunks of files from places like docker/docker's `pkg/archive/changes.go` where the Linux Foundation is clearly not the primary copyright holder.  How about:

> Copyright 2016 The Linux Foundation and contributors

with the emphasis for this file being on “and contributors” ;).

And I'm still not wild about either forking these packages (like you're doing here) or vendoring huge external dependencies (like you were doing before).  Is there really no interest inside Docker for moving this package out into a stand-alone docker/archive repository?  Or in handing that off to us to develop as opencontainers/archive?  I don't see docker/docker including github.com/opencontainers/image-tools/image to get this functionality back, but the other two solutions sound like they have a chance.